### PR TITLE
docs: add EU-1 region

### PR DESCRIPTION
**Pull request by adlerhurst**
### Definition of Ready

- [ ] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: docs/docs/legal/service-description/cloud-service-description.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ The following regions will be available when using our cloud service. This list
 
 - **Global**: All available cloud regions offered by our cloud provider
 - **Switzerland**: Exclusively on Swiss region
+- **US-1**: Exclusively on US-1 (Iowa) region
```
Comment (mffap) [inline docs/docs/legal/service-description/cloud-service-description.md:53]:
```suggestion
- **US-1**: Exclusively hosted on regions in the United States
```

File: docs/docs/legal/service-description/cloud-service-description.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ The following regions will be available when using our cloud service. This list
...
+- **EU-1**: Exclusively on EU-1 (Germany) region
```
Comment (mffap) [inline docs/docs/legal/service-description/cloud-service-description.md:54]:
```suggestion
- **EU-1**: Exclusively hosted on regions in the European Union
```

**Review by mffap — CHANGES_REQUESTED** [review]
I would not state the specific region here as this might change. Important information is in which geo-/political-regions the data is hosted. As a provider we should have the freedom to move within that region to optimize service delivery. More details can be provided on request.

**Comment by adlerhurst** [comment]
closed in favour of https://github.com/zitadel/zitadel/pull/7230

**Pull request closed without merging**